modules/openapi-generator/src/test/java/org/openapitools/codegen/InlineModelResolverTest.java

```diff
@@ -927,7 +927,7 @@ public void emptyExampleOnStringTypeModels() {
         assertEquals("#/components/schemas/EmptyExampleOnStringTypeModels", schema.getItems().get$ref());
 
         assertTrue(ModelUtils.getReferencedSchema(openAPI, schema.getItems()) instanceof StringSchema);
-        assertNull(ModelUtils.getReferencedSchema(openAPI, schema.getItems()).getExample());
+        assertSame("", ModelUtils.getReferencedSchema(openAPI, schema.getItems()).getExample());
```

Comment:
testng format is `assertSame(action, expected)`

Reply:
Switched to testng assert and flipped